replacement.js

```diff
@@ -15,11 +15,39 @@
 //2) String.replace() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
 //3) Array.join() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join
 
+
+//create emtpy array
+//selcet before word and change it to the after word
+//push new string
+//console log string
+
+
 function performReplace(str, before, after) {
-  //your code here
+
+var beforeArray = before.split('');
```

Comment:
`before` and `after` can be used directly and they are supposed to be single words so you do not need to split them. It will give you the same answer but you are doing unnecessary work.